numba/cuda/tests/cudapy/test_print.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import print_function
+import numpy
+from numba import cuda
+from numba import unittest_support as unittest
+
+
+def cuhello():
+    i = cuda.grid(1)
+    print(i, 1.234)
+
+
+def printfloat():
+    i = cuda.grid(1)
+    print(i, 23, 34.3, 321)
+
+
+def cuprintary(A):
+    i = cuda.grid(1)
+    print("A[", i, "]", A[i])
```

Comment:
If you remove the strings from this print statement, does `test_print_array` pass (and can it be not skipped?)?